src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -907,6 +907,65 @@ public override IOperation VisitWhileLoop(IWhileLoopOperation operation, int? ca
             return null;
         }
 
+        public override IOperation VisitVariableDeclarationGroup(IVariableDeclarationGroupOperation operation, int? captureIdForResult)
+        {
+            // We erase variable declarations from the control flow graph, as variable lifetime information is contained in a parallel data structure.
+            foreach (var declaration in operation.Declarations)
+            {
+                VisitVariableDeclaration(declaration, captureIdForResult);
+            }
+
+            return null;
+        }
+
+        public override IOperation VisitVariableDeclaration(IVariableDeclarationOperation operation, int? captureIdForResult)
```

Comment:
>public override IOperation VisitVariableDeclaration(IVariableDeclarationOperation operation, int? captureIdForResult) [](start = 8, length = 117)

```private void VisitVariableDeclaration(IVariableDeclarationOperation operation)```? And throw from ```public override IOperation VisitVariableDeclaration(IVariableDeclarationOperation operation, int? captureIdForResult)```

Reply:
Will do.

---
In reply to: [166783214](https://github.com/dotnet/roslyn/pull/24692#discussion_r166783214) [](ancestors = 166783214)